### ISSUE TITLE
Unexpected rounding when defining a number field

### ISSUE THREAD

**Issue by siegy22** [vuex-orm/vuex-orm#194]
All numbers are automatically rounded when setting a value to a number field like this:

```javascript
class Foo extends VuexORM.Model {
  static fields() {
    return { bar: this.number(null) }; 
  }
}

foo = new Foo({ bar: '10.6' });
foo.bar // => Number(10)
```

Here's a reproduction link: https://codepen.io/anon/pen/QBjOdR?editors=0010

I found out that the vuex-orm is doing a `parseInt` on [this line](https://github.com/vuex-orm/vuex-orm/blob/fc79dd45f0d5fe8f6b58dcc3262bf5bb56820afa/src/attributes/types/Number.ts#L35). 
IMO this is not the expected behaviour. [The Number JavaScript object is a wrapper object allowing you to work with numerical values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number). So why should it automatically round all numbers?

Proposal: Change this line to `parseFloat`, or if you would like to have backwards compatibility you'd need to create a new type of field, like float or double.

Keep up the good work, thanks!

**Comment by kiaking** [comment]
Hi! Thank you for the feedback! Yeah indeed. Let's make this floatable. I don't know why I wanted to make numbers integer :/

**Comment by kiaking** [comment]
This issue is now solved by Vuex ORM 0.25.6! 🎉 
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.25.6
